### PR TITLE
Add booth model

### PR DESCRIPTION
**Pull request by neungkl**

### CODE REVIEW

File: webserver/apps/booths/models.py [hunk 1]
```diff
@@ -0,0 +1,15 @@
+from django.db import models
+from apps.users.models import User
+
+
+class Booth(models.Model):
+    booth_number = models.CharField(verbose_name="Booth Number", max_length=5)
+    rental_free = models.IntegerField(verbose_name="Rental Free")
```
Comment (JThanat) [inline webserver/apps/booths/models.py]: rental_fee not rental_free

Comment (neungkl) [reply webserver/apps/booths/models.py]: Fixed

File: webserver/apps/booths/models.py [hunk 2]
```diff
@@ -0,0 +1,15 @@
+from django.db import models
+from apps.users.models import User
+
+
+class Booth(models.Model):
+    booth_number = models.CharField(verbose_name="Booth Number", max_length=5)
+    rental_fee = models.IntegerField(verbose_name="Rental Free")
```
Comment (JThanat) [inline webserver/apps/booths/models.py]: verbose_name="Rental Free" should be Rental Fee

Comment (neungkl) [reply webserver/apps/booths/models.py]: Fixed

**Pull request closed without merging**